### DISCUSSION TITLE
Doubt in test-loop in the video 03_Computer Vision

### DISCUSSION THREAD

**Discussion by kota2004** [mrdbourke/pytorch-deep-learning#1254] [Q&A]
In the video,
<img width="1588" height="1086" alt="image" src="https://github.com/user-attachments/assets/569f1392-4460-4ab2-b3da-b2c8de1716c1" />
here in test_loop while calculating test_accuracy don't we have to get softmax applied before getting argmax.

And also don't we need to use softmax always like in 03_Computer vision model_1 haven't used softmax instead used 2 relu layers after linear layers.
